examples/3d/transparency_3d.rs

```diff
@@ -2,11 +2,11 @@
 //! Shows the effects of different blend modes.
 //! The `fade_transparency` system smoothly changes the transparency over time.
 
-use bevy::prelude::*;
+use bevy::{prelude::*, render::view::MultiSampleLevel};
 
 fn main() {
     App::new()
-        .insert_resource(Msaa { samples: 4 })
+        .insert_resource(Msaa::from(MultiSampleLevel::Sample4))
```

Comment:
Rather than complicate this with two types, maybe it should just be a flat enum? `Msaa::from(MultiSampleLevel::Sample4)` is a mouth full. Why not just `Msaa::Sample1`?